src/main/java/org/opensearch/sdk/Extension.java

```diff
@@ -20,7 +20,7 @@
 import org.opensearch.common.settings.Setting;
 import org.opensearch.common.xcontent.NamedXContentRegistry;
 import org.opensearch.threadpool.ExecutorBuilder;
-import org.opensearch.common.settings.Settings;
+import org.opensearch.common.settings.Settings;;
```

Comment:
Added an extra semicolon here. Since the only other change to this file is whitespace you may want to just revert to the original (upstream/HEAD) version.

Reply:
done